style.css

```diff
@@ -320,3 +320,73 @@ section {
   display: flex;
   flex-direction: column;
 }
+
+.Contact-form {
+  background-color: #6070ff;
+  background-image: url("./images/headerBackgd.png");
+  background-size: cover;
+  border-radius: 60px 0 0 0;
+}
+
+form ul {
+  display: flex;
+  flex-direction: column;
+}
+
+#name,
+#mail,
+#textarea,
+form button {
+  width: 350px;
+  border-radius: 7px;
+  border: none;
+  margin: 5px;
+  padding-left: 20px;
+}
+
+#textarea {
+  padding-top: 20px;
+}
+
+#name,
+#mail {
+  height: 40px;
+}
+
+form button {
+  width: 150px;
+  height: 50px;
+  border-radius: 7px;
+  border: none;
+  margin-bottom: 50px;
+}
+
+form button:enabled {
+  background: #fff;
+  border: 1px solid  #6070ff;
+  color: #6070ff;
+}
+
+form button:disabled {
+  background: #fff;
+  border: 1px solid  #6070ff;
+  color: rgb(23, 43, 77);
+}
+
+form button:hover {
+  cursor: pointer;
+  background-color: skyblue;
+}
```

Comment:
- [ ] As requested by the previous reviewer please use the right colors for button interactions e.g. hover, enabled , active

see this image for reference 
 
![image](https://user-images.githubusercontent.com/59416136/155173619-c09a37ea-9181-4750-aca8-90bb98b990ee.png)